service-tests/github.js

```diff
@@ -165,6 +165,28 @@ t.create('(pre-)Release')
     value: Joi.string()
   }));
 
+t.create('Release Date. e.g release date|today')
+.get('/release-date/microsoft/vscode.json')
+.expectJSONTypes(Joi.object().keys({
+  name: Joi.equal('release date'),
```

Comment:
You can simplify this and the one below to e.g. `name: 'release date'`. I've gone ahead and done with the existing tests in #1127.